app/controllers/api/tasks_controller.rb

```diff
@@ -0,0 +1,66 @@
+class API::TasksController < ApplicationController
+  respond_to :json
+
+  def show
+    @tasks = Task.where(client_id: params[:client_id])
+    render json: @tasks
+  end
+  
+  def create
+    params.require(:client_id)
+    params.require(:description)
+    task = Task.new(
+        {:client_id => params['client_id'], :description => params['description']}
+    )
+    begin
+      saved = task.save!
+    rescue ActiveRecord::StatementInvalid => invalid
+      return render(:json => {:message => "Invalid task."}.to_json)
```

Comment:
What does this error message mean? wouldn't this try catch be logically doing the same thing as the else block below?